theorems/T000420.md

```diff
@@ -4,8 +4,14 @@ if:
   P000003: true
 then:
   P000169: true
+refs:
+  - doi: 10.1090/proc/13318
+    name: Each regular paratopological group is completely regular
```

Comment:
I think it's helpful to add the names of the authors in parentheses at the end of the name.  Makes it easier to remember who wrote the paper.